PKGBUILD

```diff
@@ -1,8 +1,6 @@
-# Maintainer: Jan Alexander Steffens (heftig) <heftig@archlinux.org>
-
 pkgbase=linux-starfive-visionfive2
 pkgver=5.15.0.arch1
-pkgrel=2
+pkgrel=3
 pkgdesc='Linux'
 url="https://github.com/starfive-tech/linux/tree/JH7110_VisionFive2_devel"
```

Comment:
Since the tag is being followed, the URL should not include link to the branch. Please update this with a follow up commit.

Reply:
Sorry I missed that. Pushed another commit.